src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -179,40 +194,42 @@
 								 Stretch="Uniform"
 								 Width="{StaticResource FocusAreaSize}"
 								 Height="{StaticResource FocusAreaSize}"
-								 Fill="{StaticResource Color03Brush}"
-								 Stroke="{StaticResource PrimaryColor02Brush}"
+								 Fill="{ThemeResource SelectControlToggleOnBackgroundColor}"
+								 Stroke="{ThemeResource SelectControlBackgroundColor}"
 								 IsHitTestVisible="False"
 								 StrokeThickness="0"
-								 Opacity="0"/>
+								 Opacity="0" />
 
 						<Grid x:Name="BackgroundBorder"
 							  Height="{StaticResource CheckAreaSize}"
 							  Width="{StaticResource CheckAreaSize}"
 							  Background="{TemplateBinding Background}"
 							  BorderBrush="{TemplateBinding BorderBrush}"
 							  BorderThickness="{TemplateBinding BorderThickness}"
-							  CornerRadius="{TemplateBinding CornerRadius}">
+							  CornerRadius="2">
 							<Path x:Name="HyphenGlyph"
 								  Data="M0,0L32,0 32,5.3 0,5.3z"
-								  Fill="{ThemeResource Color01Brush}"
+								  Fill="{ThemeResource SelectControlIndicatorColor}"
 								  VerticalAlignment="Center"
 								  HorizontalAlignment="Center"
 								  Stretch="Uniform"
 								  Width="12"
 								  Height="4"
 								  Opacity="0"
-								  AutomationProperties.AccessibilityView="Raw" />
+								  AutomationProperties.AccessibilityView="Raw"
+								  xamarin:Margin="0,2,0,0"/>
 
 							<Path x:Name="CheckGlyph"
 								  Data="M28.718018,0L32,3.2819897 10.666016,24.616999 0,13.951997 3.2810059,10.670007 10.666016,18.055033z"
-								  Fill="{ThemeResource Color01Brush}"
+								  Fill="{ThemeResource SelectControlIndicatorColor}"
 								  VerticalAlignment="Center"
 								  HorizontalAlignment="Center"
 								  Stretch="Uniform"
 								  Width="12"
 								  Height="12"
 								  Opacity="0"
-								  AutomationProperties.AccessibilityView="Raw" />
+								  AutomationProperties.AccessibilityView="Raw"
+								  xamarin:Margin="0,2,0,0" />
```

Comment:
On devrait pas avoir besoin de ça. C'est pourquoi?

Reply:
les icons de check et - sont pas bien aligner sur ios et android.

Reply:
Ok, please open github issue just so we remember